core/src/main/java/org/apache/accumulo/core/iterators/user/ColumnAgeOffFilter.java

```diff
@@ -43,6 +43,9 @@ public TTLSet(Map<String,String> objectStrings) {
       for (Entry<String,String> entry : objectStrings.entrySet()) {
         String column = entry.getKey();
         String ttl = entry.getValue();
+        // skip the negate option, it will cause an exception to be thrown
+        if (column.equals(NEGATE))
```

Comment:
A user might want to use this filter on a column named NEGATE. Could you add a condition `&& !TTL.equalsIgnoreCase("true") && !ttl.equalsIgnoreCase("false")`?